### PR TITLE
Bug 1217992 - Fix how the viewfinder would fade in when cold launchin…

### PR DESCRIPTION
**Pull request by aosmond**
…g camera.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=50128077933c91bc52407ca98f45869a43f29bd7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ea0a939bc8509bcdc746cf053ec4e0fc238919cf

File: apps/camera/js/views/viewfinder.js [hunk 1]
```diff
@@ -161,20 +161,18 @@ module.exports = View.extend({
...
-      this.show();
-      this.fadeTimeout = setTimeout(function() {
-        self.el.style.transitionDuration = '';
-        self.emit('fadedin');
-      }, ms);
+    this.show();
+    if (first) {
+      self.emit('fadedin');
+      // Delay to prevent the duration from applying too early
+      setTimeout(function() {
```
Comment (justindarc) [inline apps/camera/js/views/viewfinder.js]: try wrapping in `requestAnimationFrame(() => { ... })` instead (you also won't need `var self = this;` if you do that)

File: apps/camera/js/views/viewfinder.js [hunk 2]
```diff
@@ -161,20 +161,18 @@ module.exports = View.extend({
     }, this.fadeTime);
   },
 
-  fadeIn: function(ms) {
+  fadeIn: function(first) {
```
Comment (justindarc) [inline apps/camera/js/views/viewfinder.js]: maybe rename to something more descriptive, like `noFade`

Comment (justindarc) [inline apps/camera/js/views/viewfinder.js]: err.. nevermind, i think i misunderstood what this was indicating.. you could always use `firstRun` or something

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8e41655d0227e87dc48a69dcb68e66f6f36dd620

**Pull request merged**